pkg/registry/core/pod/storage/eviction.go

```diff
@@ -133,6 +136,13 @@ func (r *EvictionREST) Create(ctx context.Context, name string, obj runtime.Obje
 		return nil, errors.NewBadRequest("name in URL does not match name in Eviction object")
 	}
 
+	if eviction.DeleteOptions != nil && ptr.Deref(eviction.DeleteOptions.IgnoreStoreReadErrorWithClusterBreakingPotential, false) {
```

Comment:
is there not an validation.go validating evictions we can add thsi to?

Reply:
That's where i wanted to add it, but I didn't find any validation here https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/policy/validation/validation.go, maybe because it is a sub resource of Pod?